omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/DeletePatientFragmentController.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.coreapps.fragment.controller;
+
+import org.openmrs.Patient;
+import org.openmrs.User;
+import org.openmrs.api.APIAuthenticationException;
+import org.openmrs.api.PatientService;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.module.coreapps.CoreAppsConstants;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.openmrs.ui.framework.fragment.action.FailureResult;
+import org.openmrs.ui.framework.fragment.action.FragmentActionResult;
+import org.openmrs.ui.framework.fragment.action.SuccessResult;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class DeletePatientFragmentController {
```

Comment:
This isn't the most straightforward/readable code, but look at what happens
with this variable:
https://github.com/openmrs/openmrs-module-coreapps/blob/54a150113a1dc1d9c2e1f372d837a7f80d63a163/omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js#L39

On Thu, Apr 14, 2016 at 8:12 PM, Shreyans Sheth <notifications@github.com>
wrote:

> In
> omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/DeletePatientFragmentController.java
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20#discussion_r59822554>
> :
>
> > +
> > +import org.openmrs.Patient;
> > +import org.openmrs.User;
> > +import org.openmrs.api.APIAuthenticationException;
> > +import org.openmrs.api.PatientService;
> > +import org.openmrs.module.appui.UiSessionContext;
> > +import org.openmrs.module.coreapps.CoreAppsConstants;
> > +import org.openmrs.ui.framework.UiUtils;
> > +import org.openmrs.ui.framework.annotation.SpringBean;
> > +import org.openmrs.ui.framework.fragment.action.FailureResult;
> > +import org.openmrs.ui.framework.fragment.action.FragmentActionResult;
> > +import org.openmrs.ui.framework.fragment.action.SuccessResult;
> > +import org.springframework.transaction.annotation.Transactional;
> > +import org.springframework.web.bind.annotation.RequestParam;
> > +
> > +public class DeletePatientFragmentController {
>
> Ah I see. How do you suggest I go about generating a REST request like
> from inside...umm because I've only seen the REST endpoints..like where do
> I add the code
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20/files/c748a5bdae230f51fdb03cd30dbab9902fa65d82#r59822554>
>